File: Heat/Migrations/201509021416399_removeAddressPlant.vb
Imports System
Imports System.Data.Entity.Migrations
Imports Microsoft.VisualBasic

Namespace Migrations
    Public Partial Class removeAddressPlant
        Inherits DbMigration
    
        Public Overrides Sub Up()
            DropColumn("dbo.Plants", "Address")
            DropColumn("dbo.Plants", "StreetNumber")
            DropColumn("dbo.Plants", "City")
            DropColumn("dbo.Plants", "PostalCode")
            DropColumn("dbo.Plants", "Area")
            DropColumn("dbo.Plants", "Zone")
        End Sub
        
        Public Overrides Sub Down()
            AddColumn("dbo.Plants", "Zone", Function(c) c.String())
            AddColumn("dbo.Plants", "Area", Function(c) c.String())
            AddColumn("dbo.Plants", "PostalCode", Function(c) c.String())
            AddColumn("dbo.Plants", "City", Function(c) c.String())
            AddColumn("dbo.Plants", "StreetNumber", Function(c) c.String())
            AddColumn("dbo.Plants", "Address", Function(c) c.String())
        End Sub
    End Class
End Namespace

File: Heat.ConvertedToC#/Migrations/201509021416399_removeAddressPlant.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Linq;
using System.Xml.Linq;
using System.Diagnostics;
using System.Collections.Specialized;
using System.Configuration;
using System.Text;
using System.Text.RegularExpressions;
using System.Web;
using System.Web.Caching;
using System.Web.Mvc;
using System.Web.Mvc.Ajax;
using System.Web.Mvc.Html;
using System.Web.Routing;
using System.Web.SessionState;
using System.Web.Security;
using System.Web.Profile;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.Web.UI.WebControls.WebParts;
using System.Web.UI.HtmlControls;
using System.Data.Entity.Migrations;

namespace Heat.Migrations
{
	public partial class removeAddressPlant : DbMigration
	{

		public override void Up()
		{
			DropColumn("dbo.Plants", "Address");
			DropColumn("dbo.Plants", "StreetNumber");
			DropColumn("dbo.Plants", "City");
			DropColumn("dbo.Plants", "PostalCode");
			DropColumn("dbo.Plants", "Area");
			DropColumn("dbo.Plants", "Zone");
		}

		public override void Down()
		{
			AddColumn("dbo.Plants", "Zone", c => c.String());
			AddColumn("dbo.Plants", "Area", c => c.String());
			AddColumn("dbo.Plants", "PostalCode", c => c.String());
			AddColumn("dbo.Plants", "City", c => c.String());
			AddColumn("dbo.Plants", "StreetNumber", c => c.String());
			AddColumn("dbo.Plants", "Address", c => c.String());
		}
	}
}
